litmus-portal/graphql-server/pkg/rest_handlers/core_version_handler.go

```diff
@@ -20,11 +19,9 @@ func WorkflowHelperImageVersionHandler(c *gin.Context) {
 	version := WorkflowHelperImageVersion{Version: versionDetails}
 	versionByte, err := json.Marshal(version)
 	if err != nil {
-		log.Error(err)
 		utils.WriteHeaders(&c.Writer, http.StatusBadRequest)
-		return
+	} else {
+		utils.WriteHeaders(&c.Writer, http.StatusOK)
+		c.Writer.Write(versionByte)
 	}
```

Comment:
In case of err condition we will need to return with the badRequest status code. Might need to add the `return` statement in err case